### PR TITLE
Add announcement section to README.md

### PR DESCRIPTION
**Pull request by WadeBarnes**
- Inform people of the change to the branch structure.

### CODE REVIEW

**Comment by pSchlarb** [comment]
I think the gitpod link needs also the update.

File: docs/source/setup-dev.md [hunk 1]
```diff
@@ -58,13 +58,13 @@ create a virtualenv to work in
...
     - Press `Apply`
- 
+
 
 ## Detailed Setup
 
 ### Setup Python
 
-One needs Python 3.5 to work with the code. You can use `dev-setup/ubuntu/setup_dev_python.sh` script for quick installation of Python 3.5, pip 
+One needs Python 3.5 to work with the code. You can use `dev-setup/ubuntu/setup_dev_python.sh` script for quick installation of Python 3.5, pip
```
Comment (swcurran) [inline docs/source/setup-dev.md:67]: I thought for Ubuntu 20.04, we had to bump to a version beyond Python 3.5.

Comment (WadeBarnes) [reply docs/source/setup-dev.md:67]: We did, it's 3.8.  Docs need to be updated.  cc @pSchlarb

Comment (swcurran) [reply docs/source/setup-dev.md:67]: Part of this PR or will it follow.  I can approve if it is OK to leave this.

Comment (WadeBarnes) [reply docs/source/setup-dev.md:67]: @pSchlarb, To follow?

Comment (pSchlarb) [reply docs/source/setup-dev.md:67]: will incorporate it to the documentation changes.

**Pull request merged**